{
    "data_folder_path": "data/CVE-2023-23931",
    "pull_requests_urls": [
        "https://github.com/pyca/cryptography/pull/8230/commits/94a50a9731f35405f0357fa5f3b177d46a726ab3"
    ],
    "cve_name": "CVE-2023-23931",
    "repo": "pyca/cryptography",
    "warnings": {
        "cve": "CVE-2023-23931",
        "warnings": []
    },
    "files_extension_stats": {
        ".py": 2
    },
    "file_number": 2,
    "removed_files_number": 0,
    "added_files_number": 0,
    "nb_lines_affected": 10,
    "private_key_files": [],
    "languages": [
        "Python"
    ]
}